psmisc/tests/test.bats

```diff
@@ -0,0 +1,10 @@
+@test "pstree is expected version" {
+  expected_version="$(echo $TEST_PKG_IDENT | cut -d/ -f 3)"
+  actual_version="$(hab pkg exec $TEST_PKG_IDENT pstree --version 2>&1 | head -1 | awk '{print $3}')"
+  [ "$expected_version" = "$actual_version" ]
+}
+
+@test "pslog runs" {
+  run hab pkg exec $TEST_PKG_IDENT pslog 1
```

Comment:
This test depends on Habitat using pid 1. Is that safe?

Reply:
Pid1 is going to depend on the context this is executed in.

It might be better to use `pstree` as that supports a help and version.

`prtstat $$` might be worth running as well, that will give you the status of the parent process which _should_ be safe. We'd not be able to check the output as it will be different every time, but it should always execute successfully. I can't think of why in this context that would fail, as the parent process should be run by the same user that is executing `prtstat`. There may be some nuance I'm forgetting or not aware of that might cause it to fail on a users system, but as long as we can run it reliably in CI then I think it's a safe test.

Reply:
Thanks for the suggestion, changed to a `prtstat $$` test